src/ucp/wireup/select.c

```diff
@@ -1865,26 +1906,34 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
          * Allow selecting additional lanes in case the remote memory will not be
          * registered with this memory domain, i.e with GPU memory.
          */
+        ucp_wireup_dev_usage_count_init(&dev_count[0]);
+        bw_info.local_dev_bitmap           = UINT64_MAX;
+        bw_info.remote_dev_bitmap          = UINT64_MAX;
         bw_info.criteria.title             = "obtain remote memory pointer";
         bw_info.criteria.local_cmpt_flags |= UCT_COMPONENT_FLAG_RKEY_PTR;
         bw_info.criteria.lane_type         = UCP_LANE_TYPE_RKEY_PTR;
+        bw_info.criteria.arg               = &dev_count[0];
+        bw_info.dev_count                  = &dev_count[0];
         bw_info.max_lanes                  = 1;
 
         UCP_CONTEXT_MEM_CAP_TLS(context, UCS_MEMORY_TYPE_HOST, access_mem_types,
                                 tl_bitmap);
         ucp_wireup_add_bw_lanes(select_params, &bw_info, tl_bitmap,
-                                UCP_NULL_LANE, select_ctx, 0);
+                                select_ctx, 0);
     }
 
+    bw_info.local_dev_bitmap          = UINT64_MAX;
+    bw_info.remote_dev_bitmap         = UINT64_MAX;
```

Comment:
why dupliced these 2 lines instead of keeping them as they were?

Reply:
Because those bitmaps are updated by `add_bw_lanes` so we need to reset them after rkey_ptr.